src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -763,7 +763,7 @@ private Uri CheckProtocol(Uri uri)
 
```

Comment:
@iSazonov, your last commit had 3 failures in `PowerShell-CI-macos`
Read-Host.[Console]::ReadKey() implementation on non-Windows.Should output correctly
```powershell
Expected 0, but got 1.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Host/Read-Host.Tests.ps1: line 25
25:             $LASTEXITCODE | Should -Be 0
```
Test-Connection.Ping.Force IPv4 with implicit PingOptions
```powershell
Expected exactly '10.79.10.230', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 83
83:             $result[0].Address | Should -BeExactly $realAddress
```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.10.230', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 257
257:             $result[0].TargetAddress | Should -BeExactly $realAddress
```